source/utilities.tex

```diff
@@ -7320,12 +7340,12 @@
 contains an object of type \tcode{T}.
 
 \pnum
-A program
-that instantiates the definition of template \tcode{expected<T, E>}
-for a reference type, a function type, or
-for possibly cv-qualified types \tcode{in_place_t}, \tcode{unexpect_t}, or
-a specialization of \tcode{unexpected} for the \tcode{T} parameter
-is ill-formed.
+A type \tcode{T} is a \term{valid value type for \tcode{expected}}, if
+\tcode{remove_cv_t<T>} is \tcode{void} or a complete non-array object type
+that is not \tcode{in_place_t}, \tcode{unexpect_t}, or a specialization of
+\tcode{unexpected}. A program which instantiates class template
```

Comment:
```suggestion
\tcode{unexpected}. A program that instantiates the class template
```
```cli
$ grep 'program which' *.tex | wc -l
0
$ grep 'a program that' *.tex | wc -l
14
```

Reply:
I agree with Walter that we shouldn't make natural language decisions purely based on ngram statistics, devoid of context or rationale.

More importantly, the paper is explicitly changing prior "program that" wording to "program which", so I would hesitate to immediately overrule the wording group's explicit output here. Generally, we apply the approved wording as is, and it's mainly presentational style details (like OK/error comments, or the equivalent-to wording you pointed out elsewhere) that we adjust. For the change you propose here I would much prefer a separate follow-up issue.

Reply:
That makes sense. I should be more careful. Dismiss this. Since the current wording makes sense anyways, I'm not going to attempt pursuing this for consistency.

Reply:
Opened https://github.com/cplusplus/draft/issues/5994 for this.